.github/workflows/submitty_ci.yml

```diff
@@ -222,9 +222,9 @@ jobs:
             uses: actions/cache@v2
             with:
               path: path ~/.cache/pip
-              key: ${{ runner.os }}-py-pip-${{ hashFiles('**/requirements.txt') }}
+              key: ${{ runner.os }}-${{ github.job }}-pip-${{ hashFiles('**/requirements.txt') }}
```

Comment:
Shouldn't this be `github.sha`?

Reply:
No, the key should look like `ubuntu-e2e-pip-HASH`, so that we can restore the cache by `ubuntu-e2e-pip-`.

Reply:
Sorry, I should have specified that I mean that should it be `${{ hashFiles('**/requirements.txt') }}` -> `${{ github.sha }}`, given that we don't yet have a requirements.txt file?

Reply:
Yes, I've reverted it, this is intentional, I will update this field as I make the pull request which adds the `requirments.txt` which is almost ready.